src/lib/Review.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+
+  export let questionsToReview;
+  const dispatch = createEventDispatcher();
+
+  $: currentQuestion = questionsToReview[0];
+
+  // The <= is a catch-all in case we ever have less than one question remaining for some reason.
+  $: reviewIsAboutToComplete = questionsToReview.length <= 1;
+
+  function nextQuestion() {
+    if (reviewIsAboutToComplete) {
+      return dispatch('reviewComplete', true);
+    }
+    questionsToReview = questionsToReview.slice(1);
+  }
+</script>
+
+<div>
+  <h2 class="my-4 text-lg font-semibold">Review</h2>
+  {#key currentQuestion}
```

Comment:
Why so you need a key here?

Reply:
My bad, I had been playing with transitions and forgot to remove it.